app/index.jade

```diff
@@ -42,8 +42,8 @@ body(ng-controller="AppCtrl" ui-view="body" ng-mousemove="resetInactivityTime()"
   //- This does not depend on Angular and will show as soon as the images are done loading.
   .blocket-loading
     div
-      img.full(src="img/blocket-full.png")
-      img(src="img/blocket-no-boosters.png")
+      img.full(src="img/blocket-full.png" alt="Loading")
```

Comment:
You're right, just noticed the other examples of alt. I will update those, thanks